### PR TITLE
added rep potion,added rep potion to drop,new ico

### PR DESCRIPTION
**Pull request by X4N1**

### CODE REVIEW

File: Helbreath.HGServerConsole/BossDropConfiguration.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@ int BossDropConfiguration::GetThirdRareDrop(float fProbC)
 	case  3: if (dice(1, (2 * fProbC)) == 2) itemId = 742; break; // BagOfGold-largest
 	case  4: if (dice(1, (2 * fProbC)) == 2) itemId = 868; break; // AcientTablet(LU)
 	case  5:
-	case  6:
+	case  6: if (dice(1, (2 * fProbC)) == 2) itemId = 1139;
```
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/BossDropConfiguration.cpp]: missing break;

File: Helbreath.HGServerConsole/char/Client.h [hunk 2]
```diff
@@ -393,6 +393,7 @@ class CClient sealed : public Unit
 	bool m_bIsOnTower; 
 	bool m_bIsOnWarehouse ; 
 	bool m_bIsInBuilding; 
+	int  m_iRating;
```
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/char/Client.h]: There is `m_reputation`

File: Helbreath.HGServerConsole/HG.cpp [hunk 3]
```diff
@@ -17771,6 +17771,12 @@ void CGame::UseItemHandler(int iClientH, short sItemIndex, short dX, short dY, s
 			}
 			break;
 
+        case DEF_ITEMEFFECTTYPE_REPPLUS:
+            iMax = 10000;
+            if (m_pClientList[iClientH]->m_iRating < iMax) m_pClientList[iClientH]->m_iRating += 3;
```
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/HG.cpp]: You should use `m_reputation` that is already field of this class

File: Helbreath.HGServerConsole/char/item/Item.h [hunk 4]
```diff
@@ -138,6 +138,7 @@ enum ItemTypes
 #define ITEMEFFECTTYPE_ITEMFARM				30  
 #define ITEMEFFECTTYPE_SLATES			31
 #define ITEMEFFECTTYPE_JEWELRY			32
+#define DEF_ITEMEFFECTTYPE_REPPLUS   33
```
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/char/item/Item.h]: Let's keep it consistent with other names, and rename `DEF_ITEMEFFECTTYPE_REPPLUS` to `ITEMEFFECTTYPE_REPPLUS`

**Comment by X4N1** [comment]
.

File: Helbreath.HGServerConsole/HG.cpp [hunk 5]
```diff
@@ -17773,7 +17773,7 @@ void CGame::UseItemHandler(int iClientH, short sItemIndex, short dX, short dY, s
 
         case DEF_ITEMEFFECTTYPE_REPPLUS:
```
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/HG.cpp]: without def

File: Helbreath.HGServerConsole/char/item/Item.h [hunk 6]
```diff
@@ -138,7 +138,7 @@ enum ItemTypes
 #define ITEMEFFECTTYPE_ITEMFARM				30  
 #define ITEMEFFECTTYPE_SLATES			31
 #define ITEMEFFECTTYPE_JEWELRY			32
-#define DEF_ITEMEFFECTTYPE_REPPLUS   33
+#define DEF_ITEMEFFECTTYPE_REPPLUS         33
```
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/char/item/Item.h]: without def

File: Helbreath.HGServerConsole/HG.cpp (hunk 5 above)
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/HG.cpp]: without def

File: Helbreath.HGServerConsole/HG.cpp [hunk 7]
```diff
@@ -17771,6 +17771,12 @@ void CGame::UseItemHandler(int iClientH, short sItemIndex, short dX, short dY, s
 			}
 			break;
 
+        case DEF_ITEMEFFECTTYPE_REPPLUS:
```
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/HG.cpp]: without def

File: Helbreath.HGServerConsole/char/item/Item.h [hunk 8]
```diff
@@ -138,6 +138,7 @@ enum ItemTypes
 #define ITEMEFFECTTYPE_ITEMFARM				30  
 #define ITEMEFFECTTYPE_SLATES			31
 #define ITEMEFFECTTYPE_JEWELRY			32
+#define DEF_ITEMEFFECTTYPE_REPPLUS         33
```
Comment (pawelsawicz) [inline Helbreath.HGServerConsole/char/item/Item.h]: without def

**Pull request merged**